askbot/media/js/post.js

```diff
@@ -576,8 +576,10 @@ CommentVoteButton.prototype.setScore = function (score) {
  */
 CommentVoteButton.prototype.setVoted = function (voted) {
```

Comment:
so in `Comment.prototype.setContent` there's a line `vote.setVoted(data.upvoted_by_user)` which means that `voted` argument can be false, but previous code always set `upvoted` class no matter the value, which doesn't seem correct